backend/indexer/types.py

```diff
@@ -20,7 +22,7 @@ class DataIngestionConfig(BaseModel):
     embedder_config: EmbedderConfig = Field(
         title="Embedder configuration",
     )
-    parser_config: ParserConfig = Field(
+    parser_config: Dict[str, Union[ParserConfig, str]] = Field(
```

Comment:
What happens when I give `str` as value?

I was wondering we would just do
```suggestion
    parser_config: Dict[str, ParserConfig] = Field(
```

Reply:
This was for backward compatiblity, but now I'll be removing it.